Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/alphagov/verify/ruby:2.6.6
+FROM ruby:2.6.6
```

Comment:
Looks like this gets built in a pipeline, so will need some creds, probably with the build-task thing.
https://github.com/alphagov/verify-terraform/blob/master/environments/tools/platform/files/pipelines/verify-self-service.yml#L177